src/applications/gi/components/profile/CalculatorForm.jsx

```diff
@@ -516,6 +551,113 @@ class CalculatorForm extends React.Component {
     );
   }
 
+  renderExtensionBeneficiaryZIP() {
+    const { profile, inputs, onShowModal } = this.props;
+    const extensions = this.getExtensions();
+
+    let amountInput;
+    let extensionSelector;
+    let extensionOptions;
+    let zipcodeRadioOptions;
+
+    if (extensions.length > 0) {
```

Comment:
It looks like `extensions` could potentially be `undefined` from `getExtensions()`. Maybe check `if (extensions && extensions.length)`.